src/eos/eos.hpp

```diff
@@ -157,7 +158,7 @@ class EquationOfState {
 
  private:
   // (C++11) in-class Default Member Initializer (fallback option):
-  const Real float_min{std::numeric_limits<float>::min()};
+  const Real float_min{std::numeric_limits<Real>::min()};
```

Comment:
hmm, we need to think about this. If we want to make it `Real`, then the variable should be renamed to `real_min`

Reply:
Using "float" is fine with me so I changed to that.